### DISCUSSION TITLE
FreeSurfer 7

### DISCUSSION THREAD

**Discussion by MatthieuJoulot** [aramis-lab/clinica#583] [Show and tell]
Here are shown the results of the tests and changes to make in order to use FreeSurfer 7 with Clinica (Which currently uses FreeSurfer 6).

Tests:

- `t1-freesurfer` : tested, works; basis of the next pipelines. FreeSurfer 7 brings more files, but the one which existed in FreeSurfer 6 still do.
- `t1-freesurfer-template`: tested, works
- `t1-freesurfer-longitudinal`: tested, works
- `dwi-connectome`: tested, works
- `pet-surface`: tested, works
- `pet-surface-longitudinal`: tested, works
- `stat-surface` : tested, works.

Changes to make:

There is one change to make on `t1-freesurfer-atlas`: the additionnal atlas have a specific format in which the version is specified. Currently the version of FreeSurfer being 6, the code for the atlases is `6p0`. It'll have to be updated to `7p1`.

**Comment by omar-rifai** [comment]
Great! thanks for the effort. It might be worthwhile to also mark the outputs that have been visually compared with the current references so as to  ensure stability.
